src/parser/generated_parser.ts

```diff
@@ -3379,7 +3380,7 @@ export class GeneratedParser extends Parser {
     }
 
     @memoize
-    invalid_arguments(): any | null {
+    invalid_arguments(): never | null {
```

Comment:
`never` because it raises an exception when invalid?

Reply:
Yeah. It either returns null or throws.